### PR TITLE
Add a visualization explaining the release schedule

### PR DESCRIPTION
**Pull request by danielrozenberg**
This PR adds a page that visually shows, on a calendar (non-date-specific), how the release schedule works. For each release channel (e.g., beta, stable, lts) this page answers these two questions:

By hovering your mouse / tapping on any day in the top row, it answers the question "if I merged a code on this day, when will it be available on ?"
By hovering your mouse / tapping on any day in the bottom row, it answers the slightly more confusing question: "on this day, when was the code for merged?"

This has been moved from https://github.com/ampproject/amphtml/pull/26371, see plenty of discussion there for more context

_Slightly_ older versions of both pages, without the amp.dev shell, are deployed at https://output.jsbin.com/qovahib/quiet and https://output.jsbin.com/lofuxam/quiet

### CODE REVIEW

File: pages/content/amp-dev/documentation/guides-and-tutorials/contribute/vendor-contributions/release-schedule-accessible.html [hunk 1]
```diff
@@ -0,0 +1,176 @@
+---
+$title: Release Schedule (screen-reader version)
+---
+{% do doc.styles.addCssFile('/css/components/molecules/tip.css') %}
+{% do doc.styles.addCssFile('/css/components/organisms/release-schedule-accessible.css') %}
+{% do doc.amp_dependencies.add('amp-bind', '0.1') %}
+
+<p>For a detailed description of the various release channels, see <a href="https://github.com/ampproject/amphtml/blob/master/contributing/release-schedule.md">release-schedule.md</a>. There is also a <a href="./release-schedule.html">visual</a> (not screen-reader accessible) version of this page.</p>
```
Comment (sebastianbenz) [inline pages/content/amp-dev/documentation/guides-and-tutorials/contribute/vendor-contributions/release-schedule-accessible.html]: Would be good to add a bit more description here. 

Please link to the release schedule doc on amp.dev: `/documentation/guides-and-tutorials/learn/spec/release-schedule/?format=websites`. The links are missing margin as well:

Screen Shot 2020-01-28 at 20 45 39

File: pages/content/amp-dev/documentation/guides-and-tutorials/contribute/vendor-contributions/release-schedule.html [hunk 2]
```diff
@@ -0,0 +1,284 @@
+---
+$title: Release Schedule
+---
+{% do doc.styles.addCssFile('/css/components/organisms/release-schedule.css') %}
+{% do doc.amp_dependencies.add('amp-script', '0.1') %}
+
+<p>For a detailed description of the various release channels, see <a href="https://github.com/ampproject/amphtml/blob/master/contributing/release-schedule.md">release-schedule.md</a>. There is also a <a href="./release-schedule-accessible.html">screen-reader accessible</a> version of this page.</p>
+
+<amp-script layout="intrinsic" width="1050" height="0" script="calendar-script">
```
Comment (sebastianbenz) [inline pages/content/amp-dev/documentation/guides-and-tutorials/contribute/vendor-contributions/release-schedule.html]: Curious: why `height=0`?

Comment (danielrozenberg) [reply pages/content/amp-dev/documentation/guides-and-tutorials/contribute/vendor-contributions/release-schedule.html]: Annoying workaround - `<amp-script>` requires a `height` attribute but `layout="intrinsic"` ignores the height

File: pages/content/amp-dev/documentation/guides-and-tutorials/contribute/vendor-contributions/release-schedule-accessible.html [hunk 3]
```diff
@@ -0,0 +1,176 @@
+---
+$title: Release Schedule (screen-reader version)
```
[the remark this answers is not in the document]
Comment (danielrozenberg) [reply pages/content/amp-dev/documentation/guides-and-tutorials/contribute/vendor-contributions/release-schedule-accessible.html]: Yes, done.

File: pages/content/amp-dev/documentation/guides-and-tutorials/contribute/vendor-contributions/release-schedule-accessible.html (hunk 1 above)
Comment (danielrozenberg) [reply pages/content/amp-dev/documentation/guides-and-tutorials/contribute/vendor-contributions/release-schedule-accessible.html]: Done, hope this is a good enough description :)

**Comment by danielrozenberg** [comment]
> * mobile: font too small, also reduce left and right padding
> 
> Screen Shot 2020-01-28 at 20 49 42
> 
> * mobile: can you make the visualization take the full width?
> 
> Screen Shot 2020-01-28 at 20 49 52

These paddings come from the base style, I'm not sure if I can work around. Increased the font size for all sorts of widths, PTAL.

**Comment by matthiasrohmer** [comment]
@sebastianbenz, shall I pass this on to Longo for him to design this?

**Comment by sebastianbenz** [comment]
Would be great if he could take a look. Merging this for now because it's already super useful.

**Comment by matthiasrohmer** [comment]
Absolutely agree! Team mates asking when a specific fix will be live is one of the most common questions I get 😅

**Comment by danielrozenberg** [comment]
> @sebastianbenz, shall I pass this on to Longo for him to design this?

@matthiasrohmer I would be glad to have this looked at and improved by more capable designers than me! :)

**Pull request merged**